src/algebra/archimedean.lean

```diff
@@ -64,7 +64,7 @@ begin
   obtain ⟨k, h1, h2⟩ := exists_int_smul_near_of_pos ha g,
   rw add_gsmul at h2,
   refine ⟨k, sub_nonneg.mpr h1, _⟩,
-  simpa [sub_lt_iff_lt_add'] using h2
+  simpa [sub_lt_iff_lt_add] using h2
```

Comment:
It looks like you've still swapped these lemmas to me, based on the fact you changed this

Reply:
ok, so should this sequence of commands perform the name-switch?  I got some compilation issues after that...

```
-- replace sub_lt_iff_lt_add' with adomaniadomani
find . -type f -name "*.lean" -exec sed -i "s/sub_lt_iff_lt_add'/adomaniadomani/g" {} +

-- replace sub_lt_iff_lt_add with garbage sub_lt_iff_lt_add'
find . -type f -name "*.lean" -exec sed -i "s/sub_lt_iff_lt_add/sub_lt_iff_lt_add'/g" {} +

-- replace sub_lt_iff_lt_add with adomaniadomani sub_lt_iff_lt_add'
find . -type f -name "*.lean" -exec sed -i "s/adomaniadomani/sub_lt_iff_lt_add/g" {} +
```

Reply:
No, because the lemma is called `div_lt_iff_lt_mul` and you need to change that too!